course/src/main/java/in/testpress/course/util/FileEncryptionAndDecryption.kt

```diff
@@ -1,84 +1,77 @@
 package `in`.testpress.course.util
 
-import android.content.Context
-import android.util.Base64
-import java.io.*
-import java.security.SecureRandom
-import javax.crypto.Cipher
-import javax.crypto.KeyGenerator
-import javax.crypto.SecretKey
-import javax.crypto.spec.IvParameterSpec
-import javax.crypto.spec.SecretKeySpec
+import androidx.annotation.VisibleForTesting
+import java.io.File
+import java.io.IOException
+import java.io.RandomAccessFile
 
-class FileEncryptionAndDecryption(val context: Context) {
+class FileEncryptionAndDecryption {
```

Comment:
You are not performing encryption/decryption. You are just reversing the file. So you can write it in separate class.